src/modules/flow/persistence/persistence.c

```diff
@@ -40,8 +40,293 @@
 #include "sol-str-table.h"
 #include "sol-util.h"
 
-#include "fs-storage.h"
-#include "efivarfs-storage.h"
+#ifdef USE_FILESYSTEM
+#include "sol-fs-storage.h"
+#endif
+
+#ifdef USE_EFIVARS
+#include "sol-efivarfs-storage.h"
+#endif
+
+#ifdef USE_MEMMAP
+#include "sol-memmap-storage.h"
+#endif
+
+static bool once = true;
+
+#ifdef USE_FILESYSTEM
+static void
+test_fs(void)
```

Comment:
How difficult would it be to have at least the filesystem tests being executed by `make check`?